.gitmodules

```diff
@@ -6,8 +6,8 @@
 [submodule "contracts/lib/openzeppelin-contracts"]
 	path = contracts/lib/openzeppelin-contracts
 	url = https://github.com/openzeppelin/openzeppelin-contracts
-[submodule "subnet-evm"]
-	path = subnet-evm
-	url = git@github.com:ava-labs/subnet-evm.git
 [submodule "lib/openzeppelin-contracts"]
 	branch = v4.8.1
+[submodule "contracts/lib/subnet-evm"]
+	path = contracts/lib/subnet-evm
+	url = https://github.com/ava-labs/subnet-evm
```

Comment:
Is there a specific reason to use the https clone URL over ssh? I'm just noticing it now, but we should use the same URL type for all submodules.

Reply:
used https because rest of our submodules also use htttps, and the documentation on github tend to use https as default over ssh.